Makefile

```diff
@@ -58,7 +58,7 @@ bootstrap: pythoncheck pipcheck
 #
 # Rules for running our tests and for running various different linters
 # ###############################################
-test: lint pythoncheck tensorflowcheck
+test: lint pythoncheck
```

Comment:
@ianlivingstone what do you think about leave `tensorflowcheck` out of these three? having them got in the way when I was testing tf-int64 but maybe there's a better way that allows us to keep them?